C/Coin-OR/Bonmin/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+using BinaryBuilder, Pkg
+
+name = "Bonmin"
+version = v"1.8.8"
+
+sources = [
+    GitSource("https://github.com/coin-or/Bonmin.git",
+              "65c56cea1e7c40acd9897a2667c11f91d845bb7b"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/Bonmin
+
+if [[ ${target} == *mingw* ]]; then
+    sed -i s/dllimport/dllexport/ /workspace/destdir/include/coin-or/IpoptConfig.h
```

Comment:
This is leaving `IpoptConfig.h` in the tarball of `Bonmin`, it must be deleted at the end

Reply:
Thanks for the heads up. I'll take a look. I guess I should have paid more attention to that warning.